sum_up_zero.cpp

```diff
@@ -91,15 +92,23 @@ int main()
 	int n=6;
 	std::vector<int> s1(s,s+n); 
```

Comment:
Since you make the conversion from int[] to std::vector frequently it might be better to add that conversion in the beginning of your function call "three_sum()"